packages/cli/src/commands/translate/prompts.ts

```diff
@@ -0,0 +1,46 @@
+import { type LanguageTag } from '@logto/language-kit';
+import { conditionalString } from '@silverhand/essentials';
+
+type GetTranslationPromptProperties = {
+  sourceFileContent: string;
+  targetLanguage: LanguageTag;
+  extraPrompt?: string;
+};
+
+export const getTranslationPrompt = ({
+  sourceFileContent,
+  targetLanguage,
```

Comment:
nit: is it good to use the tag instead of the full language name (see the original code that uses `languages[languageTag]`)? since sometimes it the tag doesn't work (i met one or two times).

good to keep using tag if everything's fine when you test

Reply:
the type of `targetLanguage` is `LanguageTag`.
<img width="435" alt="image" src="https://user-images.githubusercontent.com/10806653/231944073-5a1aa745-f4e0-4492-a2d8-775b55dc8412.png">

Reply:
Update, use `language[languageTag]` in the prompt.